### PR TITLE
Updated acct_stop procedure to avoid acct_stop packet if it's duplicate

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Detect if PacketFence receive duplicates accounting stop and avoid insertion in the DB.

# Impacts
RADIUS authorize accounting

# Issue
Duplicate entries in the radacct_log table

# Delete branch after merge
YES

# NEWS file entries

## Enhancements
* Avoid duplicates accounting stop entries

### CODE REVIEW

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 1]
```diff
@@ -119,4 +119,71 @@ DROP table api_user;
...
+-- Updated freeradius acct_stop procedure
+--
+
+DROP PROCEDURE IF EXISTS acct_stop;
+DELIMITER /
+CREATE PROCEDURE acct_stop (
+  IN p_timestamp datetime,
+  IN p_framedipaddress varchar(15),
+  IN p_acctsessiontime int(12),
+  IN p_acctinputoctets bigint(20),
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 2]
```diff
@@ -119,4 +119,71 @@ DROP table api_user;
...
+  IN p_acctoutputoctets bigint(20),
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 3]
```diff
@@ -119,4 +119,71 @@ DROP table api_user;
...
+  IN p_nasporttype varchar(32),
+  IN p_acctauthentic varchar(32),
+  IN p_connectinfo_stop varchar(50),
+  IN p_calledstationid varchar(50),
+  IN p_callingstationid varchar(50),
+  IN p_servicetype varchar(32),
+  IN p_framedprotocol varchar(32),
+  IN p_acctterminatecause varchar(12),
+  IN p_acctstatustype varchar(25),
+  IN p_tenant_id int
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: int(11) unsigned

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 4]
```diff
@@ -119,4 +119,71 @@ DROP table api_user;
...
+)
+BEGIN
+  DECLARE Previous_Input_Octets bigint(20);
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 5]
```diff
@@ -119,4 +119,71 @@ DROP table api_user;
...
+  DECLARE Previous_Output_Octets bigint(20);
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 6]
```diff
@@ -119,4 +119,71 @@ DROP table api_user;
...
+  DECLARE Previous_Session_Time int(12);
```
Comment (satkunas) [inline db/upgrade-X.X.X-X.Y.Z.sql]: unsigned?

**Review by satkunas — CHANGES_REQUESTED** [review]

**Comment by fdurand** [comment]
Done

**Review by satkunas — APPROVED** [review]

**Pull request merged**